release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,11 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-oc-adm-upgrade-status-tp"]
+==== Introducing `oc adm upgrade status` command (Tech Preview)
+
+Previously, customers could retrieve information about the status of their cluster update from the web console and from the CLI. In addition to its primary focus, the `oc adm upgrade` command provided some limited information about the upgrade status. With this release, there is now a tech preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and compliments the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Comment:
Same feedback as above:
```suggestion
Previously, customers could retrieve information about the status of their cluster update from the web console and from the CLI. In addition to its primary focus, the `oc adm upgrade` command provided some limited information about the upgrade status. With this release, there is now a Technology Preview `oc adm upgrade status` command which decouples the status information from `oc adm upgrade` and compliments the UI by providing customers with specific information regarding their cluster update, including the status of the control plane and worker node updates.
```

Reply:
I still see `tech preview`,  did you forget to submit the change?

Reply:
I've bubbled this back up in [a new thread][1], since I don't have the power to un-resolve this thread, but the feedback does not seem to have been addressed.

[1]: https://github.com/openshift/openshift-docs/pull/75679#discussion_r1600680037